templates/external-dns/0/rancher-compose.yml

```diff
@@ -0,0 +1,35 @@
+.catalog:
+  name: "Route53 DNS"
+  version: "v0.1.2"
```

Comment:
We have been adding -rancherX for a build number... that way we can change the templates without incrementing the version.